controller/testTCPserver.py

```diff
@@ -0,0 +1,28 @@
+import unittest 
+from tcpServer import *
```

Comment:
Since it seems like you're only importing TcpServer, I'd just make that explicit to give the code more clarity.